setup.py

```diff
@@ -29,8 +29,8 @@ def get_static(name, condition=None):
 # scripts to be added to the $PATH
 scripts = get_static("scripts", condition="'.' in x")
 
-with open("VERSION", 'r') as versionfile:
-	version = versionfile.read().strip()
+with open("looper/_version.py", 'r') as versionfile:
```

Comment:
I left it this way because the way in place was file-parsing-based. From what I could glean, there's a lack of consensus. I have absolutely no preference about this. If placing we want to go the placing it in `__init__.py` and then import as you mention, I'm totally on board with that. Mostly, I want to have these changes in master and the 0.4 version finalized so that I can stop worrying about mege conflicts arising.